packages/run-utils/ava.config.js

```diff
@@ -1,8 +1,11 @@
-import baseConfig from '../../ava.base.js'
-
 const config = {
```

Comment:
Why not extend config from the base config?

Reply:
because each package has it's own flaws and things (like the disabling of the workers) and the test splitting. Each package has its own config and it's not needed to extend from something (just adds a layer of complexity)